lib/generators/rodauth/install_generator.rb

```diff
@@ -5,6 +5,9 @@ module Rodauth
   module Rails
     module Generators
       class InstallGenerator < ::Rails::Generators::Base
+        # source_root "#{__dir__}/templates"
+        # destination_root = "#{__dir__}/templates"
+
```

Comment:
Why comment this out?

Reply:
> Why comment this out?
I wasn't sure exactly of what they were doing, and to alert you to their existence.